src/tree/GroupTreeItemBase.ts

```diff
@@ -124,14 +129,14 @@ export class GroupTreeItemBase extends AzExtParentTreeItem {
             localize('showingResources', 'Showing {0} of {1} resources. Click to reveal all resources.', numOfResources, numOfTotalResources) :
             localize('hideAllResources', 'Click to hide filtered resources.');
```

Comment:
Not directly related to this PR but I noticed this when it was originally added. I don't _love_ this string, but I'm not sure what else works better. My thinking is that if the resource was filtered, it would _already_ be hidden, not in need of hiding.

I thought of "Click to hide unsupported resources" but "unsupported" seems like it might be alarming if misunderstood, so I don't like that either.

Reply:
Yeah, I'm waiting for @fiveisprime to sprinkle some PMagic

![image](https://user-images.githubusercontent.com/5290572/166523265-d38f37e4-7524-4d99-bd64-597d89246139.png)

Because I definitely agree.

Reply:
We're feeling the media today aren't we

![everywhere](https://c.tenor.com/zkcKxJWzfZYAAAAd/gifs-everywhere-toys-story.gif)